api/test/service/upload.test.js

```diff
@@ -46,18 +52,17 @@ test('upload/add inserts into DB mapping between data CID and car CIDs', async (
   const root = car.roots[0]
   const shards = [car.cid, otherCar.cid]
 
-  /** @type {import('../../service/types').UploadItemOutput[]} */
   const uploadAdd = await UploadCapabilities.add.invoke({
     issuer: alice,
     audience: uploadService,
     with: spaceDid,
     nb: { root, shards },
     proofs: [proof]
-    // @ts-expect-error ʅʕ•ᴥ•ʔʃ
   }).execute(connection)
 
-  // @ts-expect-error error is added by ucanto if it fails
-  t.not(uploadAdd.error, true, uploadAdd.message)
+  if (uploadAdd.error) {
+    throw new Error(uploadAdd.message)
```

Comment:
IDK might be better to keep the whole error response than pick out the message?

```suggestion
    throw new Error('invocation failed', { cause: uploadAdd })
```

Reply:
Yeah, I will add this in all of them 👍🏼